ydb/library/yql/dq/opt/bitset.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <stdlib.h>
+
+/* 
+ * This header contains helper functions for working with bitsets.
+ * They are templated by TNodeSet, which is a std::bitset<>.
```

Comment:
+ We use the the template for efficiency: for 64 bit nodesets we implement a faster next subset functionality